posts/2020-11-30-open-liberty-devfile-stack.adoc

```diff
@@ -0,0 +1,84 @@
+
+== Try it out!
+
+After link:https://odo.dev/docs/installing-odo/[installing ODO], you can use the Open Liberty stack to either link:https://github.com/OpenLiberty/application-stack/wiki/Using-the-Default-Starter#getting-started[create a new Open Liberty starter application] or initialize an existing app. For this blog, we will be using the link:https://github.com/OpenLiberty/application-stack-intro[application-stack-intro] project as a starting point.
+
+
+[source,sh]
+----
+$ git clone git@github.com:OpenLiberty/application-stack-intro.git && \
+  cd application-stack-intro
+----
+
+This application is a basic JAX-RS sample using Maven, Open Liberty and Microprofile. 
+You will also notice an additional `devfile` packaged with the app. This is the brains behind the Open Liberty stack and outlines how the application will be built, run and deployed using the stack's development image. 
+
+For convenience and illustration, this devfile is included with the intro app; however, there is nothing app specific in the devfile. If one was not included, the same devfile would be automatically pulled down from the odo devfile registry when initialized with odo. No changes necessary. 
+
+Log into your OpenShift cluster and create an odo component.
+
+[source,sh]
+----
+   $ odo create my-ol-component
+----
+
+The application-stack-intro app is now initialized with odo and ready to deploy. 
+
+[source,sh]
+----
+   $ odo push
+----
+
+Just like that, the app is running in your cluster. No need to build an image, create a deployment or run any custom commands. The stack and ODO does everything behind the scenes.
+
+== Start coding!
+
+Not only is the app running in OpenShift, it is running in link:https://openliberty.io/blog/2019/10/22/liberty-dev-mode.html[dev mode] ready to pick up any changes you make. Each time you make a change, you can either re-issue `odo push` to sync your local changes with your remote cluster, or simply have odo watch for changes automatically.
+
+[source,sh]
+----
+   $ odo watch
+----
+
+To access the app, the Open Liberty stack creates a default URL for your deployment named `ep1`. You can find that in the URLs list. 
+
+[source,sh]
+----
+   $ odo url list
+----
+
+Open a broswer to this URL and append `/api/resource` for the REST endpoint. You should see:
+
+[source,text]
+----
+   Hello! Welcome to Open Liberty
+----
+
+Now let's make a change to the REST API. Open `src/main/java/dev/odo/sample/StarterResource` and change the `getRequest()` String response to "Open Liberty is awesome!"
+
+Refresh your browser. The application running in OpenShift is automatically updated!
+
+== Learn more
+
+For more on ODO, check out: https://odo.dev
+
+For more details on the Open Liberty stack, check out: https://github.com/OpenLiberty/application-stack/wiki
```

Comment:
What about the CP4A KCtr ?   Is this worth linking to?   You might ask Laura or YK for general guidelines in OL -> CP4A refs.